docs/user-guide/managing-deployments.md

```diff
@@ -117,8 +117,8 @@ It is a recommended practice to put resources related to the same microservice o
 A URL can also be specified as a configuration source, which is handy for deploying directly from configuration files checked into github:
 
 ```console
-$ kubectl create -f https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/docs/user-guide/replication.yaml
-replicationcontrollers/nginx
+$ kubectl create -f https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/docs/user-guide/pod.yaml
```

Comment:
Because  docs/user-guide/replication.yaml will be deleted by this PR.  And Changing the URL doesn't change the meaning of the example.